.storybook/polyfills.js

```diff
@@ -1,3 +1,7 @@
+// Used solely for storybook development
+import '@babel/polyfill';
```

Comment:
I’d suggest maintaining a separate polyfills list for Story-book env-specific polyfills, like [one in vanilla](https://github.com/carbon-design-system/carbon-components/blob/b282853/demo/polyfills/devenv.js). Also, I’d suggest staying away from importing the entire Babel polyfill as doing so will deprive our way to test against IE11 for the list of polyfills.

Reply:
> importing the entire Babel polyfill

It's important to note the `babel-preset-env` config set to `'usage'` to help out with this situation.

Reply:
Babel is eager to bring in polyfills - I believe you have been surprised how many Babel polyfills was brought in for a simple case, when we discussed this topic before?